Jint.Tests/Runtime/InteropTests.cs

```diff
@@ -56,6 +55,39 @@ public void PrimitiveTypesCanBeSet()
             ");
         }
 
+        [Fact]
+        public void CanAccessMemberNamedItem()
+        {
+            _engine.Execute(@"
+                    function item2(arg) {
+                        return arg.item2
+                    }
+                    function item1(arg) {
+                        return arg.item
+                    }
+                    function item3(arg) {
+                        return arg.Item
+                    }
+            ");
+
+            var argument = new Dictionary<string, object>
+            {
+                {"item2", "item2 value"},
+                {"item", "item value"},
+                {"Item", "Item value"}
+            };
+
+            Assert.Equal("item2 value", _engine.Invoke("item2", argument));

```

Comment:
Is this just to ensure that it's not breaking dictionary indexers?

Reply:
Yes I wanted to make sure that normal objects (non-dictionaries) still allow both indexing and using Item/item property (preferring real Item property over the indexer).